hoover/site/events.py

```diff
@@ -78,16 +82,9 @@ def on_auto_logout(sender, username, **kw):
         def on_login_failure(sender, otp_failure, **kw):
             save(type='loginFailed', otp_failure=otp_failure)
 
-    if installed.ratelimit:
-        from ..contrib.ratelimit import signals as ratelimit_signals
-
-        @receiver(ratelimit_signals.rate_limit_exceeded)
-        def on_rate_limit_exceeded(sender, counter, request, **kw):
-            save(
-                type='forceLogout',
-                key=counter.key,
-                username=request.user.get_username(),
-            )
+        @receiver(twofactor_signals.rate_limit_exceeded)
```

Comment:
Are we limiting normal (non-2fa) logins too? If not, would it make sense to limit them too?